homeassistant/data_entry_flow.py

```diff
@@ -207,6 +207,13 @@ async def async_create_flow(
         Handler key is the domain of the component that we want to set up.
         """
 
+    async def async_flow_aborted(
```

Comment:
Note that this is *only* called if a flow is already aborted after a step method is done.
It's not called if the step returns abort.

Reply:
https://github.com/home-assistant/core/pull/142138/files#r2032327983

Reply:
No longer relevant after the approach is changed